Build error where memcpy is not declared in scope

**Describe your environment** 
Devcontainer with ubuntu 21.04, cmake 3.21.2 , GCC 9.3.0 and with CPM as dependency manager.

**Steps to reproduce**
Building application with BaggagePropagator cause opentelemetry build error where _memcpy_ is not declared in scope.

In file included from /workspaces/<project_name>/build/_deps/opentelemetry-cpp-src/api/include/opentelemetry/baggage/baggage.h:8,

from /workspaces/<project_name>/build/_deps/opentelemetry-cpp-src/api/include/opentelemetry/baggage/propagation/baggage_propagator.h:6,

from /workspaces/<project_name>/standalone/source/main.cpp:3:
/workspaces/<project_name>/build/_deps/opentelemetry-cpp-src/api/include/opentelemetry/common/kv_properties.h: In member function 'opentelemetry::v1::nostd::unique_ptr<const char []> opentelemetry::v1::common::KeyValueProperties::Entry::CopyStringToPointer(opentelemetry::v1::nostd::string_view)':
/workspaces/<project_name>/build/_deps/opentelemetry-cpp-src/api/include/opentelemetry/common/kv_properties.h:191:7: error: 'memcpy' was not declared in this scope

**What is the expected behavior?**
Able to build the project :)

**What is the actual behavior?**
Error

**Additional context**

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -16,6 +16,7 @@ Increment the:
 ## [Unreleased]
 
 * Support environment variables for both `OtlpGrpcExporter` and `OtlpHttpExporter` ([#983](https://github.com/open-telemetry/opentelemetry-cpp/pull/983))
+* Fix build issue where _memcpy_ was not declared in scope ([#985](https://github.com/open-telemetry/opentelemetry-cpp/issues/985))
 
 ## [1.0.0] 2021-09-16
 

--- a/api/include/opentelemetry/common/kv_properties.h
+++ b/api/include/opentelemetry/common/kv_properties.h
@@ -11,6 +11,7 @@
 #include "opentelemetry/nostd/unique_ptr.h"
 #include "opentelemetry/version.h"
 
+#include <cstring>
 #include <string>
 #include <type_traits>
 

